### ISSUE TITLE
SIRI GeneralMessage : ItemIdentifier et InfoMessageIdentifier

### ISSUE THREAD

**Issue by AntoineAugusti** [etalab/transport-profil-siri-fr#9]
![image](https://github.com/etalab/transport-normes/assets/295709/f3d0415a-b8b1-4a42-a875-16ef0d3e40e9)

Qu'elle est la différence ou l'utilité de ces 2 champs dans une réponse `GeneralMessage` ? Ils semblent très similaires.

[Côté CEN](https://github.com/SIRI-CEN/SIRI/blob/master/examples/siri_exm_GM/exm_generalMessage_response.xml) 🇪🇺 je ne vois pas d'attribut `ItemIdentifier`.

**Comment by albanpeignier** [comment]
Je suis d'accord sur le manque de clarté de la définition. Dans la pratique, nous sommes arrivés à ces usages dans Ara :

- InfoMessageIdentifier est l'identifier du GeneralMessage ("le modèle métier"). C'est le même qui est utilisé durant toute la vie du dit-modèle.
- ItemIdentifier est simplement l'identifiant technique de ce "message"/cette notification SIRI, de cette payload SIRI. C'est utilisé aussi dans d'autres structures SIRI avec la même sémantique (VehicleMonitoring, etc, etc). C'est une information purement technique (... qui est ignoré pour les GMs par Ara par exemple).

Par exemple, pour un message "sample", le flux SIRI pourra ressembler à :

```html
<siri:GeneralMessage>
  <!-- ... -->
  <siri:ItemIdentifier>0bbfc11b-d8a4-11ef-b822-18c04dede996</siri:ItemIdentifier>
  <siri:InfoMessageIdentifier>sample</siri:InfoMessageIdentifier>  
</siri:GeneralMessage>
``` 

puis un deuxième message qui met à jour le même message :

```html
<siri:GeneralMessage>
  <!-- ... -->
  <siri:ItemIdentifier>0fb969ff-d8a4-11ef-93c1-18c04dede996</siri:ItemIdentifier>
  <siri:InfoMessageIdentifier>sample</siri:InfoMessageIdentifier>  
</siri:GeneralMessage>
```

**Comment by TuThoThai** [comment]
Je clos ce ticket car résolu. En revanche, je garde le marquage `FAQ` pour une future consolidation.
